tests/utils/test_proto_json_utils.py

```diff
@@ -106,18 +110,22 @@ def assert_result(result, expected_result):
     # With schema
     schema = Schema(
         [
-            TensorSpec(np.dtype("object"), [-1], "a"),
+            TensorSpec(np.dtype("str"), [-1], "a"),
             TensorSpec(np.dtype("float32"), [-1], "b"),
             TensorSpec(np.dtype("int32"), [-1], "c"),
         ]
     )
+    dfSchema = Schema([ColSpec("string", "a"), ColSpec("float", "b"), ColSpec("integer", "c"),])
```

Comment:
Nit : move this after the comment on line 126.
```suggestion
    dfSchema = Schema([ColSpec("string", "a"), ColSpec("float", "b"), ColSpec("integer", "c")])
```